internal/pkg/errors/errors.go

```diff
@@ -147,6 +151,14 @@ func (e *ActivateServiceAccountError) Error() string {
 	return FAILED_SERVICE_ACCOUNT_ACTIVATION
 }
 
+type SetInexistentProfile struct {
+	Profile string
+}
+
+func (e *SetInexistentProfile) Error() string {
+	return fmt.Sprintf(SET_INEXISTENT_PROFILE, e.Profile, e.Profile)
```

Comment:
You don't need to pass the argument here twice, you can pass it once and use `%[1]s` (you can see some examples in the other errors)